code/game/machinery/excelsior/redirector.dm

```diff
@@ -0,0 +1,160 @@
+
+/obj/machinery/excelsior_redirector/proc/doReboot(mob/living/carbon/human/user)
+	if(user.stat || user.incapacitated() || !user.Adjacent(src))
+		return
+	to_chat(user, SPAN_NOTICE("You start rebooting \the [src] with new information. Your hands start moving by themselves like they're remotely guided to input new information."))
+	if(do_after(user, 15 SECONDS, src))
+		to_chat(user, SPAN_NOTICE("You succesfully reboot \the [src]. Your hands are no longer moving on their own."))
+		deltimer(rebootTimer)
+		var/datum/faction/excelsior/commies = get_faction_by_id(FACTION_EXCELSIOR)
+		for (var/datum/antagonist/A in commies.members)
+			to_chat(A.owner.current, SPAN_EXCEL_NOTIF("\The [src] has been rebooted by [user]. It will need another reboot in 3 minutes."))
+		addtimer(CALLBACK(src, PROC_REF(requestReboot)),3 MINUTES)
+		icon_state = "redirector_running"
+	else
+		to_chat(user, SPAN_DANGER("Your hands suddenly stop moving.. \the [src] wasn't rebooted."))
+
+/obj/machinery/excelsior_redirector/proc/tryRuin(mob/living/carbon/human/user)
+	if(user.stat || user.incapacitated() || !user.Adjacent(src))
+		return
+	to_chat(user, SPAN_NOTICE("You start bending \the [src]'s antenna! Its quite tough..."))
+	var/datum/faction/excelsior/commies = get_faction_by_id(FACTION_EXCELSIOR)
+	for (var/datum/antagonist/A in commies.members)
+		to_chat(A.owner.current, SPAN_EXCEL_NOTIF("The [src]'s antenna is being bent by someone! Stop them."))
+	if(do_after(user, 1 MINUTE, src))
+		stopRedirecting()
+		antennaBent = TRUE
+		icon_state = "redirector_bent"
+		/*
+		var/datum/faction/excelsior/commies = get_faction_by_id(FACTION_EXCELSIOR)
+		for (var/datum/antagonist/A in commies.members)
+			to_chat(A.owner.current, SPAN_NOTICE("The [src]'s antenna has been bent! Progress has been reset and it needs to be fixed!"))
+		*/
+	else
+		to_chat(user, SPAN_NOTICE("You stop bending the antenna, it snaps back to its original form."))
+
+/obj/machinery/excelsior_redirector/proc/finishRedirecting()
+	icon_state = "redirector_finished"
+	flick(icon, "redirector_finishing")
+	spawn(12 SECONDS)
+		SSticker.excelsior_hijacking = 2
+
+/obj/machinery/excelsior_redirector/proc/stopRedirecting()
+	deltimer(redirectTimer)
+	if(rebootTimer)
+		deltimer(rebootTimer)
+	SSticker.excelsior_hijacking = 0
+	var/decl/security_state/security_state = decls_repository.get_decl(GLOB.maps_data.security_state)
+	security_state.set_security_level(oldSecurityLevel, force_change = TRUE)
+	var/datum/faction/excelsior/commies = get_faction_by_id(FACTION_EXCELSIOR)
+	for (var/datum/antagonist/A in commies.members)
+		to_chat(A.owner.current, SPAN_EXCEL_NOTIF("\The [src]'s redirect has been stopped! You have failed the commune!"))
+		var/mob/living/carbon/human/unworthy = A.owner.current
+		var/obj/item/implant/excelsior/implnt = locate(/obj/item/implant/excelsior/) in unworthy
+		if(implnt)
+			implnt.execute()
```

Comment:
This is a bit too extreme, as extreme as someone summoning a stash fuck-off nowhere.

Reply:
For maintainers , 
![image](https://github.com/discordia-space/CEV-Eris/assets/61350382/e446502f-4d1f-440a-a383-5100dedc95ca)

Reply:
What if the implants just broke, alongside most Excel machinery?